gossip/privdata/pvtdataprovider.go

```diff
@@ -137,7 +138,9 @@ func (ec *eligibilityComputer) computeEligibility(pvtdataToRetrieve []*ledger.Tx
 			eligibleMissingKeys[key] = rwsetInfo{
 				invalid: invalid,
 			}
-
+			if invalid {
+				numInvalid++
+			}
```

Comment:
Before this for loop, we've already broken out if we're not supposed to store private data of invalid transactions, therefore this would only get incremented if we're supposed to store private data of invalid transactions. Is that what you intended?

Reply:
storePvtdataOfInvalidTx is a separate flag that's tied to the 2.0 capability. skipPulllingInvalidTx is configured on a peer level